pkg/lca/imagebasedupgrade.go

```diff
@@ -289,6 +289,22 @@ func (builder *ImageBasedUpgradeBuilder) AutoRollbackOnFailureInitMonitorTimeout
 	return builder
 }
 
+// AutoRollbackOnFailureDisabledInitMonitor allows disabling the watchdog
+// triggering a rollback upon upgrade failure within the set timeout.
+// Set to false by default.
+func (builder *ImageBasedUpgradeBuilder) AutoRollbackOnFailureDisabledInitMonitor(
+	flag bool) *ImageBasedUpgradeBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Setting the Init Monitor for Auto Rollback on failure to %b in imagebasedupgrade", !flag)
```

Comment:
Now you're really trying to make me scratch my head 😆

Reply:
Exactly the pain we all share with this :)

So the log would be:
"Setting the Init Monitor for Auto Rollback on failure to true in imagebasedupgrade" if we set
Spec.AutoRollbackOnFailure.DisabledInitMonitor to false